src/contracts/core/AllocationManager.sol

```diff
@@ -219,6 +277,65 @@ contract AllocationManager is
         _setAllocationDelay(msg.sender, delay);
     }
 
+    /// @inheritdoc IAllocationManager
+    function setAVSRegistrar(
+        IAVSRegistrar registrar
+    ) external {
+        _avsRegistrar[msg.sender] = registrar;
+        emit AVSRegistrarSet(msg.sender, getAVSRegistrar(msg.sender));
+    }
+
+    /// @inheritdoc IAllocationManager
+    function updateAVSMetadataURI(
+        string calldata metadataURI
+    ) external {
+        emit AVSMetadataURIUpdated(msg.sender, metadataURI);
+    }
+
+    /// @inheritdoc IAllocationManager
+    function createOperatorSets(
+        CreateSetParams[] calldata params
+    ) external {
+        for (uint256 i = 0; i < params.length; i++) {
+            OperatorSet memory operatorSet = OperatorSet(msg.sender, params[i].operatorSetId);
+
+            // Create the operator set, ensuring it does not already exist
+            require(_operatorSets[msg.sender].add(operatorSet.id) == true, InvalidOperatorSet());
```

Comment:
can remove the == true

Reply:
[61772f2](https://github.com/Layr-Labs/eigenlayer-contracts/pull/860/commits/61772f2f930aa2a816f4ee3a1eb6416e505f8596)